app/uk/gov/hmrc/incometaxpenaltiesappealsfrontend/controllers/SingleAppealConfirmationController.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
+
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.config.ErrorHandler
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.controllers.predicates.{AuthAction, UserAnswersAction}
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.services.UserAnswersService
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.views.html._
+import uk.gov.hmrc.incometaxpenaltiesfrontend.controllers.predicates.NavBarRetrievalAction
+import uk.gov.hmrc.incometaxpenaltiesappealsfrontend.pages.SingleAppealConfirmationPage
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+
+class SingleAppealConfirmationController @Inject()(singleAppealConfirmationView: SingleAppealConfirmationView,
+                                      val authorised: AuthAction,
+                                      withNavBar: NavBarRetrievalAction,
+                                      withAnswers: UserAnswersAction,
+                                      userAnswersService: UserAnswersService,
+                                      override val errorHandler: ErrorHandler,
+                                      override val controllerComponents: MessagesControllerComponents
+                                     )(implicit ec: ExecutionContext) extends BaseUserAnswersController {
+
+
+//  def onPageLoad(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers).async { implicit user =>
+//      Future(Ok(singleAppealConfirmationView("first", 300.00)))
+//  }
+
+  def onPageLoad(): Action[AnyContent] = (authorised andThen withNavBar andThen withAnswers) { implicit user =>
+
+    user.penaltyData.multiplePenaltiesData match {
+      case Some(multiplePenaltiesData) =>
+        Ok(singleAppealConfirmationView(
+          //TODO" add first/second appeal dynamically
+          "first",
```

Comment:
Can't pass `"first"` here, because that won't get translated to `Welsh`

We probably need two different message keys for LPP1 and LPP2, that is determined by whether `user.isAdditional` is `true` or `false`